components/Textbox.tsx

```diff
@@ -1,39 +1,50 @@
-import { View } from "react-native";
+import { NativeSyntheticEvent, TextInputKeyPressEventData, View } from "react-native";
 import { ThemedText } from "./ThemedText";
 import { getTextEventCommand } from "@/hooks/protocolBuffer";
 import { PiConnectionProps } from "./Connection";
+import { TextInput } from "react-native-gesture-handler";
 
 
-function getCodeForEvent(e: React.KeyboardEvent): number {
-  switch (e.key) {
+const SEND_ENTER_PROTOCOL_PACKET = getTextEventCommand('\n'.charCodeAt(0));
+
+function getCodeForEvent(e: NativeSyntheticEvent<TextInputKeyPressEventData>): number {
+  switch (e.nativeEvent.key) {
     case "Enter":
```

Comment:
nit: not sure if we spent a lot of time looking into whether we can eliminate this case for the `"Enter"` key.